pylint_django/tests/input/func_unused_arguments.py

```diff
@@ -0,0 +1,19 @@
+"""
+Checks that Pylint still complains about unused-arguments if a function/method
+contains an argument named `request`.
+"""
+# pylint: disable=missing-docstring
+
+from django.http import JsonResponse
+from django.views import View
+
+
+def user_detail(request, user_id):  # [unused-argument]
+    # nothing is done with user_id
+    return JsonResponse({'username': 'steve'})
+
+
+class UserView(View):
+    def get(self, request, user_id):  # [unused-argument]
+        # nothing is done with user_id
+        return JsonResponse({'username': 'steve'})
```

Comment:
Both of these are already covered in other test files:

- FBV: [`func_noerror_views.py`](https://github.com/PyCQA/pylint-django/blob/85430c526ba24076e9b30ca191438312318f0501/pylint_django/tests/input/func_noerror_views.py)
- CBF: [`func_noerror_classviews.py`](https://github.com/PyCQA/pylint-django/blob/85430c526ba24076e9b30ca191438312318f0501/pylint_django/tests/input/func_noerror_classviews.py)

I can merge these test files into the new test file if you want to. I didn't do that originally because the examples aren't producing any errors and I though it would be better to leave them in the `func_noerror...` files.

Reply:
Right, good catch. I didn't see that. 

Maybe only for the sake of visibility just copy the existing code (or my comment or your variant, up to you) into the existing test file. If you feel like it add a comment explaining why the copy.  If you don't want to bother I can copy that too. 

Othewise +1 for the tests as-is.

Reply:
Done. I have copied the examples from your comment.